docs/advanced/input_files/input-main.md

```diff
@@ -3431,4 +3438,37 @@ for `nspin 2` case. The difference is that `lambda`, `target_mag`, and `constrai
 
 - **Default**: none
 
+[back to top](#full-list-of-input-keywords)
+
+## Quasiatomic Orbital (QO) analysis
+
+These variables are used to control the usage of QO analysis.
+
+### qo_switch
+
+- **Type**: Boolean
+- **Description**: whether to let ABACUS output QO analysis required files
+- **Default**: 0
+
+### qo_basis
+
+- **Type**: String
+- **Description**: specify the type of atomic basis
+  - `pswfc`: use the pseudowavefunction in pseudopotential files as atomic basis. To use this option, please make sure in pseudopotential file there is pswfc in it.
+  - `hydrogen`: generate hydrogen-like atomic basis, whose charge is read from pseudopotential files presently.
+- **Default**: `hydrogen`
+
+### qo_screening_coeff
+
+- **Type**: Real
```

Comment:
Why is the definition of `qo_screening_coeff`  a vector<double> instead of double?

Reply:
I will explain in this doc